.about.yml

```diff
@@ -0,0 +1,72 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: openFEC-web-app
+
+# Full proper name of the project (required)
+full_name: openFEC-web-app
```

Comment:
No, this shouldn't appear on the dashboard--@mbland added a whitelist so that only pre-specified projects appear in the dashboard, not all projects with an .about.yml. That said, I'm happy to change the name here if folks would like to.